### ISSUE TITLE
Minimum and optimal android device specifications

### ISSUE THREAD

**Issue by WillemvdW** [justadudewhohacks/face-api.js#143]
Do you have some guidance of the minimum and optimal device specification to run face-api.js offline on a mobile device to detect faces, and to do facial recognition

**Comment by justadudewhohacks** [comment]
Not really. It should run on newer devices for sure, I was even able to run it on my old android phone. They tiny face detector and the landmark nets are very mobile friendly.

**Comment by ScottDellinger** [comment]
@justadudewhohacks In the same line of thought... are there any current device restrictions you can think of? I know it won't run in IE11, but are there any other known issues?

**Comment by ScottDellinger** [comment]
I've tested it in Chrome (PC and Android), Firefox (PC and Android), Safari (on both Macbook and iPhone) and all seem to work! IE definitely doesn't work, as it doesn't support async functions.

**Comment by justadudewhohacks** [comment]
Hmm, but your talking about the examples here, right? The source code of the library itself gets transpiled down to es5, which should run in IE11.

**Comment by rover886** [comment]
@justadudewhohacks 

Is there any possibility that this should work on offline system?

I am working on a chromium based project for school examination software, where student will give exam on a computer which is not connected to internet, exam software will be deployed in local LAN. So if I integrate this to detect face, will it work if there is no internet? Can prefetch the model (and any other files) needed to run this.
